Makefile

```diff
@@ -50,7 +50,7 @@ generate-licenses-cli-json:
 	cargo about generate --all-features --format json -o all_licenses_cli.json
 
 generate-licenses-desktop:
-	cargo about generate --all-features -m $(DESKTOP_DIR)/Cargo.toml $(DESKTOP_DIR)/about.hbs -o all_licenses_desktop.html
+	cargo about generate --all-features -m $(DESKTOP_DIR)/Cargo.toml $(DESKTOP_DIR)/about.hbs -o nym-vpn-desktop/public/licenses-rust.html
```

Comment:
Imho let's make the same path change to the json build rule below.

Also, since I see that you added public/licenses-rust.html, we might as well remove the license file that is currently in the root dir